artifacts/contracts/mocks/exchange-rate-precompile/ExchangeRatePrecompileMock.sol/ExchangeRatePrecompileMock.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
If the system contract mocks are all moved into `./test/foundry` along with the `/utils` then these artifacts won't even be generated by hardhat.

Reply:
Let's do it